home/src/main/resources/rdf/abox/filegraph/dynamic-api-individuals.n3

```diff
@@ -20,66 +20,129 @@
+                                dynapi_java:validators.IsMaxLength ;
+        rdfs:label              "Is max length"@en-US ;
+        vitro:mostSpecificType  dynapi:validator .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/validator/IsMaxValue>
+        a                       dynapi:validator ,
+                                dynapi_java:validators.Validator ,
+                                dynapi_java:validators.IsMaxValue ;
+        rdfs:label              "Is max value"@en-US ;
+        vitro:mostSpecificType  dynapi:validator .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/validator/IsMinValue>
+        a                       dynapi:validator ,
+                                dynapi_java:validators.Validator ,
+                                dynapi_java:validators.IsMinValue ;
+        rdfs:label              "Is min value"@en-US ;
+        vitro:mostSpecificType  dynapi:validator .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/validator/IsRegex>
+        a                       dynapi:validator ,
+                                dynapi_java:validators.Validator ,
+                                dynapi_java:validators.IsRegex ;
+        rdfs:label              "Is regular expression"@en-US ;
+        vitro:mostSpecificType  dynapi:validator .
+
 #-------------------------------------------------------------------------------
 #
 # Parameter types
 #
 
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/type/XSDboolean>
+        a                       dynapi:parameterType ,
+                                dynapi_java:ParameterType ;
+        dynapi:typeName         "boolean" ;
+        rdfs:label              "Boolean type"@en-US ;
+        vitro:mostSpecificType  dynapi:parameterType .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/type/XSDdecimal>
+        a                       dynapi:parameterType ,
+                                dynapi_java:ParameterType ;
+        dynapi:typeName         "decimal" ;
+        rdfs:label              "Decimal type"@en-US ;
+        vitro:mostSpecificType  dynapi:parameterType .
+
 <https://vivoweb.org/ontology/vitro-dynamic-api/parameter/type/XSDinteger>
         a                       dynapi:parameterType ,
-                                dynapi_java:ParameterType;
-        dynapi:typeName         "integer" ;        
+                                dynapi_java:ParameterType ;
+        dynapi:typeName         "integer" ;
         rdfs:label              "Integer type"@en-US ;
         vitro:mostSpecificType  dynapi:parameterType .
 
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/type/XSDindividual>
+        a                       dynapi:parameterType ,
+                                dynapi_java:ParameterType ;
+        dynapi:typeName         "individual" ;
+        rdfs:label              "Individual type"@en-US ;
+        vitro:mostSpecificType  dynapi:parameterType .
```

Comment:
Not sure whether we will need separated parameter types for researchers, organizations, etc., or this generic one will be enough. @litvinovg any opinion on this?